### ISSUE TITLE
Switch pointsinside to use Rvcg::vcgClostKD

### ISSUE THREAD

**Issue by jefferis** [jefferis/nat#352]
This should fix some cases where points are declared to be outside a mesh.

**Comment by jefferis** [comment]
Example:

```r
v2a=fetchn_fafb('name:MBON a2sc Right ASB', mirror=FALSE)
plhl=as.mesh3d(FCWBNP.surf,c('SLP_L','SIP_L','LH_L'))
v2alhonly=pointsinside(v2a, plhl)
points3d(xyzmatrix(v2a), col=ifelse(v2alhonly, 'red', 'black'))
plot3d(FCWBNP.surf, mat=c("SIP_L", "SLP_L", "LH_L"), col="red", alpha=0.2)
```

**Comment by jefferis** [comment]
Closed by caf26fa
